BRFCore/Code/UI/UIColor+BR.m

```diff
@@ -0,0 +1,67 @@
+//
+//  UIColor+BR.m
+//  BlueRocketFuel
+//
+//  Created by Estevan Hernandez on 11/2/16.
+//  Copyright © 2016 Blue Rocket. All rights reserved.
+//
+
+#import "UIColor+BR.h"
+
+@implementation UIColor (BR)
++ (UIColor *)colorFromRGBHex:(uint64_t)rgb {
```

Comment:
The implementations create `CGColorRef` objects, only to turn them into `UIColor`. What about creating `UIColor` directly, so there is less to maintain? I'm again thinking of [BRStyle's implementation](https://github.com/Blue-Rocket/BRStyle/blob/1a1d13a8b9880a800c69379c9d963e5fcef24c61/BRStyle/Code/Core/BRUIStyle.m#L82-L119).